Gruntfile.js

```diff
@@ -20,12 +20,24 @@ module.exports = function (grunt) {
   var JS                    = 'target/js',
       JASMINE_TEST_FOLDER   = 'build2/reports/jasmine',
       JASMINE_TEST_FILE     = JASMINE_TEST_FOLDER + '/login.html',
-      JSHINT_OUT_FILE       = 'build2/loginjs-checkstyle-result.xml',
+      ESLINT_OUT_FILE       = 'build2/loginjs-eslint-checkstyle.xml',
       DIST                  = 'dist',
       SASS                  = 'target/sass',
       SCSSLINT_OUT_FILE     = 'build2/loginscss-checkstyle-result.xml',
       WIDGET_RC             = '.widgetrc',
-
+      JS_LINT_FILES         = [
+        'Gruntfile.js',
+        'src/*.js',
+        'src/**/*.js',
+        '!src/vendor/*.js',
+        '!src/util/countryCallingCodes.js',
+        'buildtools/**/*.js',
+        '!buildtools/r.js',
+        'test/unit/helpers/**/*.js',
+        'test/**/**/*.js',
```

Comment:
I didn't realize that js test files are changed from `test/**/*.js` to `test/unit/**/*.js`, now it's good to review and merge.